rpcs3/Emu/Cell/Modules/cellMic.cpp

```diff
@@ -514,7 +514,7 @@ error_code cellMicInit()
 	return CELL_OK;
 }
 
-error_code cellMicEnd(ppu_thread& ppu)
+error_code cellMicEnd(ppu_thread&)
```

Comment:
can't you remove the whole param here and in similar places?

Reply:
I don't know, sometimes it can be used as a non-intrusive disambiguator for functions with similar names.

Reply:
Plus sometimes they are declared in headers as well so I didn't want to change headers.

Reply:
Which leaves literally 3 functions where it can be easily removed, done.